kinbot/kb.py

```diff
@@ -176,17 +179,35 @@ def main():
         logging.info('Starting the search for homolytic scission products')
         well0.homolytic_scissions = HomolyticScissions(well0, par, qc)
         well0.homolytic_scissions.find_homolytic_scissions()
-  
     # initialize the master equation instance
-    mess = MESS(par, well0)
-    mess.write_input()
-    #mesmer = MESMER(par, well0)
-    #mesmer.write_input()
+
+    # check to see if uq analysis is turned on
+    if par.par['me'] == 1:
+        logging.info('ME turned on')
+        if uq == 0:
```

Comment:
Why do you have this if/elif statement? The stuff in them seem to be identical.
(I tried to select the whole block for comment but it said that my comment is pending, not sure why)

Reply:
You're right, the first if statement is repetitive, the second if statement should stay though to decide if UQ is on or off.